webapp/app/index.html

```diff
@@ -5,13 +5,13 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>Accent</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Since we’re making sure self-closing `<link>` tags have a `/>`, we could apply it to `<meta>` tags too.